git_machete/cmd.py

```diff
@@ -1345,11 +1359,13 @@ def print_line_prefix(b_, suffix):
                     if sha == fp_sha(b):
                         # fp_branches_cached will already be there thanks to the above call to 'fp_sha'.
                         fp_branches_formatted = " and ".join(map(star(lambda lb, lb_or_rb: lb_or_rb), fp_branches_cached[b]))
-                        fp_suffix = (RED + ' ➔ fork point ???' + ENDC + ' commit ' + short_sha(fp_sha(b)) + ' found in reflog of ' + fp_branches_formatted)
+                        fp_suffix = (colored(" " + right_arrow() + " fork point ???", RED) + " commit " + short_sha(fp_sha(b)) + " found in reflog of " + fp_branches_formatted)
                     else:
                         fp_suffix = ''
-                    print_line_prefix(b, u"│ " + ENDC + dim(msg) + fp_suffix + "\n")
-            print_line_prefix(b, u"└─")
+                    print_line_prefix(b, vertical_bar() + " " + ENDC + dim(msg) + fp_suffix + "\n")
+            node_ascii_only = {DIM: "|-", RED: "x-", GREEN: "o-", YELLOW: "?-"}
+            node = u"└─" if not ascii_only else node_ascii_only[edge_color[b]]
```

Comment:
Nit: I'd call it e.g. `elbow`... `node` generally has many meanings in CS, can be confusing here a bit IMO

Reply:
And `elbow_ascii_only` above